staging/src/k8s.io/api/admissionregistration/v1alpha1/types.go

```diff
@@ -190,6 +207,21 @@ type Validation struct {
 	Reason *metav1.StatusReason `json:"reason,omitempty" protobuf:"bytes,3,opt,name=reason"`
 }
 
+// AuditAnnotation describes how to produce an audit annotation for an admission request.
+type AuditAnnotation struct {
+	// Key specifies the audit annotation key.
+	// Required.
```

Comment:
document the allowed length/format of keys, and that they have to be unique

Reply:
also document how this gets combined with the policy name to make the effective annotation key, so people can actually find their annotations

Reply:
and document behavior in conflict cases between identically named webhook/admission keys

Reply:
Done!